grpc-spring-boot-starter-gradle-plugin/src/main/resources/grpc-spring-boot.gradle

```diff
@@ -40,11 +40,23 @@ sourceSets.configureEach{s->
 
 protobuf {
     protoc {
-        artifact = "com.google.protobuf:protoc:${grpcSpringBoot.protocVersion.get()}"
+
+        // for apple m1, please add protoc_platform=osx-x86_64 in $HOME/.gradle/gradle.properties
```

Comment:
On Apple M1 this project currently does not build due to missing artifacts. This is a recommended workaround. I can remove this again if you want.

Reply:
If the platform  can't be inferred dynamically, please add it as configuration option [here](https://github.com/LogNet/grpc-spring-boot-starter/blob/master/grpc-spring-boot-starter-gradle-plugin/src/main/java/org/lognet/springboot/grpc/gradle/GrpcSpringBootExtension.java)+ [readme](https://github.com/LogNet/grpc-spring-boot-starter/blob/master/grpc-spring-boot-starter-gradle-plugin/README.adoc)

Reply:
This is just a workaround and should not be needed as a property of this Spring boot starter. In case we still want to have it I'd suggest to make it as part of another pull request.

In the meantime, I will revert these changes to the `grpc-spring-boot.gradle` file.

Reply:
Yes, this is not starter's feature, but gradle plugin's one. You can have this logic in extension  class and `grpc-spring-boot.gradle` in separate PR.